reference-implementation/lib/ReadableStreamBYOBReader-impl.js

```diff
@@ -17,6 +18,9 @@ class ReadableStreamBYOBReaderImpl {
     if (view.buffer.byteLength === 0) {
       return promiseRejectedWith(new TypeError('view\'s buffer must have non-zero byteLength'));
     }
+    if (CanTransferArrayBuffer(view.buffer) === false) {
```

Comment:
This one can also be changed to `IsDetachedBuffer`. We need to change `ReadableByteStreamControllerPullInto` to do a `try..catch` around [this line](https://github.com/whatwg/streams/blob/5da8d5c2bfdde4e3ce34ec3e63f4bd3f86922390/reference-implementation/lib/abstract-ops/readable-streams.js#L1175), and then immediately call `readIntoRequest.errorSteps()`.